build/meson/tests/meson.build

```diff
@@ -183,6 +185,8 @@ test('test-zstream-1',
 test('test-zstream-3',
   zstreamtest,
   args: ['--newapi', '-t1', ZSTREAM_TESTTIME] + FUZZER_FLAGS,
+  # --newapi dies on Windows with "exit status 3221225477 or signal 3221225349 SIGinvalid"
+  should_fail: host_machine_os == os_windows,
```

Comment:
what if the issue is fixed, and `zstreamtest --newapi` now completes successfully ?
would it be considered a test failure due to `should_fail` ?

Reply:
This PR changes its status to EXPECTEDFAIL, which is a success status. When the bug is fixed, it will start producing an UNEXPECTEDPASS status, which is considered a failure and causes the overall test results to be negative.

Reply:
> This PR changes its status to EXPECTEDFAIL, which is a success status. When the bug is fixed, it will start producing an UNEXPECTEDPASS status, which is considered a failure and causes the overall test results to be negative.

Better fix the issue then

Reply:
Yep, once that other PR is merged I will re-check this and drop the obsolete commit.